src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/wrapper/ManageHearingsWrapper.java

```diff
@@ -22,7 +23,7 @@
 @NoArgsConstructor
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class ManageHearingsWrapper {
-
+    private YesOrNo processOrderAddHearing;
```

Comment:
You haven't used this yet.

Remove it?  Considering what we've discussed lately, it could possibly be made more abstract to cover multiple events.

Reply:
Stand down.  Just seen reason in defs repo.